common/include/es0_power_manager.h

```diff
@@ -23,12 +23,17 @@ enum core_error_t {
 
 /**
  * @brief Register a user of a ES0
- * @return  -1 If too many users
+ * @param baudrate Baudrate used in host side will be passed to LL. All instances must
+ * 		   use same so once set it can only be changed by stopping all instances
+ * 		   first and then reinitialize with new value
+ * @return  0 If successful
```

Comment:
```suggestion
 * @retval  0 If successful
```
and add `@retval` to other values as well.